### DISCUSSION TITLE
Some ideas in my mind

### DISCUSSION THREAD

**Discussion by dhruva-sundararajan** [aadhithya/rajiniPP#25] [General]
Hello

You can use these dialouges also as a next update if you wish to

`Varavendiya Nerathla Correct a vanduduven` - exceptions (from Muthu)
`Black Sheep` - for cases where you find string element in an integer array (from Enthiran)

Just an idea

Thanks & Regards,
Dhruva

**Comment by aadhithya** [comment]
Hi, thanks for your suggestions but at this moment there are no plans to support exceptions or arrays(due to lack of time). What is in the pipeline is behind the scene improvements to the language and features such as variable scopes and the ability to pass arguments to functions. 

If you have the time,  you can always add these features and raise a PR from your fork!
